docs/source/conf.py

```diff
@@ -0,0 +1,97 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(os.path.join("..", "..", "src")))
+
+version = "main"
+
+# -- Project information -----------------------------------------------------
+
+project = "MaRDA Metadata Extractors"
+copyright = "2021 - 2023, MaRDA Metadata Extractors Team"
+author = "Peter Kraus"
+release = version
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "sphinx.ext.autodoc",
+    # "sphinx.ext.coverage",
+    "sphinx.ext.napoleon",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.autosectionlabel",
+    # "sphinx.ext.autosummary",
+    # "sphinx_autodoc_typehints",
+    "sphinx_rtd_theme",
+    "myst_parser",
+    "sphinxcontrib.mermaid",
+    # "sphinxcontrib.autodoc_pydantic",
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ["_templates"]
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = ["_build"]
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "sphinx_rtd_theme"
+html_css_files = [
+    "custom_theme.css",
+]
```

Comment:
```suggestion
]
html_logo = "https://avatars.githubusercontent.com/u/74017645?s=200&v=4"
html_favicon = "https://www.marda-alliance.org/wp-content/uploads/2023/04/cropped-favicon-32x32.png"
```

Use the MaRDA GH org's avatar as the logo and steal the favicon from marda-alliance.org